Backend/Program.cs

```diff
@@ -10,7 +12,25 @@ private static void Main(string[] args)
 
     private static void RunServer(ServerStartupOptions serverOptions)
     {
-        Console.WriteLine("Hello, World!");
         Console.WriteLine($"Server should start on port {serverOptions.Port}");
```

Comment:
This line should be changed to something similar to "Server is up and listening on port #", and it should be printed after the server is started